### PR TITLE
get pixel size from fourcc to calculate pixel stride

### PR DESCRIPTION
**Pull request by maztheman**
implementing the TODO in buffer.c

### CODE REVIEW

**Comment by maztheman** [comment]
vncdo didnt exist in archlinux...hmm odd.

File: src/buffer.c [hunk 1]
```diff
@@ -91,9 +91,9 @@ struct wv_buffer* wv_buffer_create_shm(int width,
 	if (!self->wl_buffer)
 		goto shm_failure;
 
-	// TODO: Get the pixel size from the format instead of assuming it's 4.
+	int bpp = pixel_size_from_fourcc(fourcc);
 	self->nvnc_fb = nvnc_fb_from_buffer(self->pixels, width, height, fourcc,
-			stride / 4);
+			stride / bpp);
```
Comment (Consolatis) [inline src/buffer.c:96]: Just passing by: this divides by zero if no format matches.

Comment (maztheman) [reply src/buffer.c:96]: Suppose an assert(bpp > 0); could be added.

**Comment by any1** [comment]
Merged externally. Thanks!

**Pull request closed without merging**